ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-commands.sh

```diff
@@ -13,4 +13,7 @@ UNSAFE_CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
 SAFE_CLUSTER_NAME=${UNSAFE_CLUSTER_NAME#"ci-op-"}
 echo "${SAFE_CLUSTER_NAME}" > ${SHARED_DIR}/CLUSTER_NAME
 
-
+# Get the clouds.yaml based on the cluster type.
+if [ -f "/var/run/cluster-secrets/${CLUSTER_TYPE}/clouds.yaml" ]; then
+  cp "/var/run/cluster-secrets/${CLUSTER_TYPE}/clouds.yaml"  "${SHARED_DIR}/clouds.yaml"
+fi
```

Comment:
This should not be in an if statement. If for some reason the "cp" command should not succeed, the step should exit with the error, not continue since the clouds.yaml file will not be avialable.